src/components/Breadcrumb/components/BreadcrumbElement.js

```diff
@@ -6,7 +6,11 @@ import { ElementType } from '../types'
 /**
  * A breadcrumb element.
  */
-const BreadcrumbElement = ({ element }) => <Link href={element.url}>{element.name}</Link>
+const BreadcrumbElement = ({ element }) => (
```

Comment:
you can destructure more ? 
 ` = ({ element: { url, onClick, name}}) =>`

Reply:
less readable I think

Reply:
+1 less readable, LGTM